integration/test-integration/src/test/resources/dockerCompose/git-service.yaml

```diff
@@ -0,0 +1,11 @@
+
+  gitlab:
+    image: "gitlab/gitlab-ee:latest"
```

Comment:
Just a query, Is it possible to use, gitlab community edition docker image, maybe with a specific version?

Reply:
Yes, we can use the community edition also.